CONTRIBUTING.md

```diff
@@ -909,6 +910,34 @@ test cases.
 you should name it ``module_name_example`` (ie: `cms_form` and `cms_form_example`).
 In this way coverage analysis will ignore this extra module by default.
 
+### Investigating Travis Test Failures
+
+It can sometimes be difficult to reproduce a Travis test failure locally due to
+subtle environment differences. In these scenarios it can be helpful to connect to
+the Runbot container generated for that branch/PR via SSH, where the
+environment will be very similar to Travis. You can do this by running:
+
+```
+ssh -p [port] odoo@runbot[1 or 2].odoo-community.org
```

Comment:
What about use [Port Forwarding](https://help.ubuntu.com/community/SSH/OpenSSH/PortForwarding)?
 - `-L 18080:localhost:18069`

In order to allow open a browse http://localhost:18080/ to connect new instance